src/Sarif/NotYetAutoGenerated/Location.cs

```diff
@@ -0,0 +1,237 @@
+// Copyright (c) Microsoft.  All Rights Reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System;
+using System.CodeDom.Compiler;
+using System.Collections.Generic;
+using System.ComponentModel;
+using System.Numerics;
+using System.Runtime.Serialization;
+
+using Newtonsoft.Json;
+
+namespace Microsoft.CodeAnalysis.Sarif
+{
+    /// <summary>
+    /// A location within a programming artifact.
+    /// </summary>
+    [DataContract]
+    [GeneratedCode("Microsoft.Json.Schema.ToDotNet", "1.1.0.0")]
+    public partial class Location : PropertyBagHolder, ISarifNode
```

Comment:
>Location [](http://example.com/codeflow?start=25&length=8)

created a copy of the Location.cs to 
NotYetAutoGenerated folder to make modification.

Reply:
added